src/dcc/RailcomBroadcastDecoder.cxx

```diff
@@ -82,15 +85,15 @@ bool RailcomBroadcastDecoder::process_data(const uint8_t *data, unsigned size)
         {
             case dcc::RMOB_ADRLOW:
                 if (currentL_ == payload) {
-                    if (countL_ < REPEAT_COUNT) ++countL_;
+                    if (countL_ < MAX_REPEAT_COUNT) countL_ += 2;
```

Comment:
Please place if statement body in braces on its own line.

Reply:
fixed